packages/next/server/web/sandbox/context.ts

```diff
@@ -134,6 +134,17 @@ async function createModuleContext(options: ModuleContextOptions) {
 
         if (typeof input === 'object' && 'url' in input) {
           return __fetch(input.url, {
+            method: input.method,
+            body: input.body,
+            cache: input.cache,
+            credentials: input.credentials,
+            integrity: input.integrity,
+            keepalive: input.keepalive,
+            mode: input.mode,
+            redirect: input.redirect,
+            referrer: input.referrer,
+            referrerPolicy: input.referrerPolicy,
+            signal: input.signal,
```

Comment:
![not great, not terrible](https://pbs.twimg.com/media/ETxAjEqXQAIlplQ.jpg)

we could also `...input` but it feels harsh (there are some things like `text()` that might pop in as well, so cherry picking seems okay but also very wasteful and redundant)

Reply:
I can write one quick